src/pkg/projects/windowsdesktop/src/windowsdesktop.depproj

```diff
@@ -3,17 +3,60 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <ItemGroup>
```

Comment:
Are you doing the closure testing here that we did in DN-T?  That's important since WinForms and WPF don't communicate their dependencies and you need to satisfy them with PackageReferences here.

Reply:
No, looks like that's implemented at https://devdiv.visualstudio.com/DevDiv/_git/DotNet-Trusted?path=%2Fpkg%2FframeworkPackage.targets&version=GBmaster (and https://github.com/dotnet/corefx/blob/master/pkg/frameworkPackage.targets). I'll take a look at adding it here.

Reply:
Done, and to confirm it works I did a few builds with sections of workarounds and some CoreFX PackageReferences removed. Each thing produces expected validation errors. Nice. 😄

I didn't go through the PackageReferences systematically, but I noticed that removing `System.Security.AccessControl` and `System.Security.Cryptography.ProtectedData` passes... suggesting they're not needed. I'm not going to remove these with this PR in case it's a testing gap and would fail at runtime. Filed https://github.com/dotnet/core-setup/issues/6337 to follow up.

Reply:
Those might be transitive dependencies of other packages.  In DN-T we had to flatten the tree since we were including files 1 by 1.  Here we are including packagereferences which include dependencies transitively, which could be the reason for those removals not triggering anything.

Reply:
Aha, that makes sense. Added that to https://github.com/dotnet/core-setup/issues/6337.